local-cli/generator-android/templates/src/app/build.gradle

```diff
@@ -18,6 +19,10 @@ apply plugin: "com.android.application"
  *   // whether to bundle JS and assets in debug mode
  *   bundleInDebug: false,
  *
+ *   // whether to bundle JS and assets in another build variant
+ *   // bundleInStageAlpha: true,
+ *   // bundleInBeta: true,
```

Comment:
We should add a bit more info here, like explaining the properties, and may be a link on how to configure build variants,

e.g. -

```
// whether to bundle JS and assets in a build variant (if you have build variants configured, see <link>)
// the configuration property is in the format `bundleIn${productFlavor}${buildType`
// bundleInStageAlpha: true,
// bundleInBeta: true,
```